wrangler-core/src/main/java/io/cdap/wrangler/executor/RecipePipelineExecutor.java

```diff
@@ -127,6 +127,7 @@ public List<Row> execute(List<Row> rows) throws RecipeException {
         // Resets the scope of local variable.
         if (context != null) {
           context.getTransientStore().reset(TransientVariableScope.LOCAL);
+          context.getTransientStore().set(TransientVariableScope.LOCAL, "dq_failure", 0L);
```

Comment:
Please add a comment why this variable is being initialized here.

Reply:
Done.